### PR TITLE
MR-667 Adds Cypress tag search test

### PR DESCRIPTION
**Pull request by wavebeem**
- [x] This change is documented in CHANGELOG.md

### CODE REVIEW

**Review by tarakreft — APPROVED** [review]

File: cypress/integration/tag-search.spec.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    );
+    // Close overlay
+    cy.get("[data-testid=meridian--private--close-overlay]").click();
+    // Open tags list
+    cy.get("[data-testid=meridian--private--tag-control]").click();
+    // Search for "Chad"
+    cy.get("[data-testid=meridian--private--overlay-search]").type("Chad");
+    // Click "Chad"
+    cy.get("[data-testid=meridian--private--overlay-tag-546C0E014866]").click();
+    // Assert data in the information popup for "Chad"
```
Comment (vercasson) [inline cypress/integration/tag-search.spec.ts:27]: 👍

File: cypress/integration/tag-search.spec.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "exist"
+    );
+    cy.get("[data-testid=meridian--private--overlay-tag-546C0E032A98]").should(
+      "exist"
+    );
+    cy.get("[data-testid^=meridian--private--overlay-tag-]").should(
+      "have.length",
+      2
+    );
+    // Close overlay
```
Comment (vercasson) [inline cypress/integration/tag-search.spec.ts:19]: I think clearing the input (after searching) and asserting the expected total count would be nice also.

**Review by vercasson — APPROVED** [review]

File: cypress/integration/tag-search.spec.ts (hunk 2 above)
Comment (wavebeem) [reply cypress/integration/tag-search.spec.ts:35]: Good idea! I added a bunch more testing in line with your request

Comment (vercasson) [reply cypress/integration/tag-search.spec.ts:35]: Cool! Curious, did you try `.type("")` to clear the input first? I did 😛 .

**Pull request merged**